profiles/car.lua

```diff
@@ -358,6 +358,7 @@ function way_function(way, result)
   Handlers.run(handlers,way,result,data,profile)
 end
 
+-- Called during edge-based-graph creation
```

Comment:
Hm maybe remove impl. detail comment here, I think this might just confuse users who have no idea about the C++ side of things.